tests/aws_utils.py

```diff
@@ -178,6 +180,36 @@ def __exit__(
         self._item.delete()
 
 
+class ValidationItem:
+    def __init__(  # pylint: disable=too-many-arguments
+        self,
+        asset_id: str,
+        result: ValidationResult,
+        details: JsonObject,
+        url: str,
+        check_type: str,
```

Comment:
Same as elsewhere, we might want to call this `check_name` or just `check`. We don't have check *types* (yet).